src/mlpack/core/optimizers/sgd/sgd.hpp

```diff
@@ -135,6 +136,9 @@ class SGD
   //! Modify whether or not the individual functions are shuffled.
   bool& Shuffle() { return shuffle; }
 
+  //! Get the update policy. 
+  UpdatePolicy GetUpdatePolicy() const { return updatePolicy; }
```

Comment:
That's a good idea. So, I should make the following updates in SGD class, right? 
* `updatePolicy` to `updatePolicyType`
* `typename UpdatePolicy = VanillaUpdate` to `typename UpdatePolicyType = VanillaUpdate`
* `GetUpdatePolicy()` to `UpdatePolicy()`